public/DELETE_THIS_FILE_TOO

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is this ? 😆 
The name suggest something.
Actually delete this file

Reply:
lol... I couldn't find the delete button and thought it might be permission issue. 

I want to ask about an idea I have, but I have not tried it, if I delete the content and name of the file will GitHub delete the file automatically?

Reply:
Yes github will delete it automatically.

Reply:
Sweet 😊... let me do it